From: Sven Roederer <devel-sven@geroedel.de>
Date: Sun, 7 Jun 2020 23:23:55 +0200
Subject: olsrd: mark as broken

In October 2018 upstream started looking for a new maintainer and was going into
"freeze" mode. No maintainer was found by February 2020 and the code has gone
into "hard freeze", which in fact changed it to completely unmaintained.
Since February 2020 there is also issue #547, which is also stalled.

For this reason we mark this package as broken, till a new maintainer is found
or it gets dropped completely.

Signed-off-by: Sven Roederer <devel-sven@geroedel.de>

diff --git a/olsrd/Makefile b/olsrd/Makefile
index 5f4e1e3e176e3e382066d461311d74446afca434..83b16dc28c36598b3db6c2c5b3158db95f9e104c 100644
--- a/olsrd/Makefile
+++ b/olsrd/Makefile
@@ -36,7 +36,7 @@ endef
 define Package/olsrd
   $(call Package/olsrd/template)
   MENU:=1
-  DEPENDS:=+libpthread
+  DEPENDS:=+libpthread @BROKEN
 endef
 
 define Package/olsrd/conffiles
